packages/adaptive-web-components/src/components/picker-list/picker-list.styles.ts

```diff
@@ -14,13 +15,14 @@ export const templateStyles: ElementStyles = css`
         flex-wrap: wrap;
         outline: none;
         user-select: none;
+        gap: ${densityControl.horizontalGap};
```

Comment:
These are fine additions but need to be in `aestheticStyles` instead or migrated to style modules. Inputs have been a little strange because I was trying to make them look good when filled with an autocomplete value. I can see why Picker might be even more strange and not follow the same pattern. All `templateStyles` need to be plain, no style decisions, just structural conveniences like flex layout, etc. Eventually all `aestheticStyles` will migrate to Adaptive UI and be removed.

Reply:
moved